examples/cdk/README.md

```diff
@@ -1,14 +1,21 @@
-# Welcome to your CDK TypeScript project!
+# AWS Lambda Powertools (TypeScript) examples in CDK
 
-This is a blank project for TypeScript development with CDK.
+This is a deployable CDK app that deploys AWS Lambda functions as part of a CloudFormation stack. These Lambda functions use the utilities made available as part of AWS Lambda Powertools (TypeScript) to demonstrate their usage.
 
-The `cdk.json` file tells the CDK Toolkit how to execute your app.
+The example functions, located in the `lib` folder, are triggered automatically when deployed using the CDK construct defined in `lib/example-function.ts`.
```

Comment:
Minor suggestions:
- replace triggered with "invoked"
- specify that they're invoked twice

The reason why I think the second point is relevant is that it'll allow customers to see different logs/metrics/annotations on segments based on when the execution had a cold start or not.